### DISCUSSION TITLE
方便提供英文直接替換用的壓縮檔嗎？

### DISCUSSION THREAD

**Discussion by sif941025** [GpointChen/FFXIVChnTextPatch-GP#399] [General]
希望可以有載入高畫質，windows用，覆蓋英文的（高畫質不強求只是想測試

因為我是使用Steam deck那個exe用不太了
使用Proton開啟 輸入遊戲路徑（使用XIVlauncher）他會要求選擇正確的遊戲檔案(路徑錯誤

請求
000000.win32.dat0 
000000.win32.index 
000000.win32.index2 
0a0000.win32.dat0 
0a0000.win32.index 
0a0000.win32.index2
壓縮檔

**Comment by GpointChen** [comment]
https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.2.0.2/ffxiv-20231027-en.zip

**Reply by sif941025** [reply]
非常感謝
